client/src/components/History/services.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { safePath } from "utils/redirect";
+import { rethrowSimple } from "utils/simple-error";
+import { getQueryDict } from "store/historyStore/model/filtering";
+
+const getQueryString = (filterText) => {
```

Comment:
This should also be moved to `filter.js`, its already used in `components/History/model/queries.js`.